src/main/java/com/moodybluez/enterprise/dao/IMoodDAO.java

```diff
@@ -0,0 +1,16 @@
+package com.moodybluez.enterprise.dao;
+
+import com.moodybluez.enterprise.dto.Mood;
+
+import java.util.Map;
+
+public interface IMoodDAO {
```

Comment:
Created DAO Interfaces to be used with @AutoWired (Polymorphism optimization)

Reply:
Ditto above... looks very good; will look even better with JavaDoc.